frontend/packages/portal-frontend/src/compound/components/tiles/TopDatasetDependencies.tsx

```diff
@@ -58,9 +60,11 @@ export const TopDatasetDependencies: React.FC<TopDatasetDependencyProps> = ({
                     <p style={{ paddingLeft: "12px" }} />
                   )}
                   <a
+                    className={styles.ellipsisStyle}
                     href={`${urlPrefix}/${featureType}/${datasetCor.other_dimension_label}`}
                     target="_blank"
                     rel="noopener noreferrer"
+                    title={datasetCor.other_dimension_label}
```

Comment:
I noticed recently that we have a Tooltip component that @rcreasi uses beautifully in `frontend/packages/@depmap/data-explorer-2/src/components/DataExplorerPage/components/plot/integrations/GeneTea/GeneTeaTable.tsx`

I wonder if this would be a good place to use the tooltip component instead of title 🤔